configure

```diff
@@ -297,6 +297,11 @@ parser.add_option('--with-lttng',
     dest='with_lttng',
     help='build with Lttng (Only available to Linux)')
 
+parser.add_option('--with-lto',
+    action='store_true',
```

Comment:
Tiny nit: It would be better we passed `default=False` as one of the arguments to `add_option`.